### DISCUSSION TITLE
MS SQL 2016

### DISCUSSION THREAD

**Discussion by km37500** [directus/directus#5146] [Archive]
Hi,

will directus work with MS SQL 2016 or only with MS SQL 2019 ?

**Comment by rijkvanzanten** [comment]
We've built / tested it on 2019 primarily. I'm not too familiar with the intricacies of the changes between 2016 and 2019, so I'm a little hesitant to conclusively say if it will work or not 🤔

**Reply by martijnboland** [reply]
Yes, SQL Server 2016 will work fine. I think 2012 will work too, because that's when paging with offset and fetch was introduced.
